### PR TITLE
fix dependency/assembly problems in build

### PR DESCRIPTION
**Pull request by davidangb**
\<your comments for this PR go here\>

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [x] I've followed [the instructions](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [x] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [x] I've updated the [FISMA documentation](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [x] Get two thumbsworth of review and PO signoff if necessary
- [x] Verify all tests go green
- [x] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -36,11 +38,13 @@ object Dependencies {
 
     "org.elasticsearch.client"       % "transport"           % "5.4.3",
 
-    "com.google.api-client"          % "google-api-client"   % "1.22.0"
-      exclude("com.google.guava", "guava-jdk5"),
-    "com.google.apis"                % "google-api-services-storage" % "v1-rev58-1.21.0",
-    "com.google.apis"                % "google-api-services-compute" % "v1-rev120-1.22.0",
-    "com.google.apis"                % "google-api-services-sheets"  % "v4-rev493-1.23.0",
+    "com.google.guava"               % "guava"               % "19.0",
```
Comment (davidangb) [inline project/Dependencies.scala:41]: we could probably get away with a more recent version; this is the same one rawls uses

File: project/Dependencies.scala [hunk 2]
```diff
@@ -36,11 +38,13 @@ object Dependencies {
...
+
+    excludeGuavaJDK5("com.google.api-client"          % "google-api-client"   % "1.23.0"),
+    excludeGuavaJDK5("com.google.apis"                % "google-api-services-storage" % "v1-rev116-1.23.0"),
+    excludeGuavaJDK5("com.google.apis"                % "google-api-services-compute" % "v1-rev163-1.23.0"),
+    excludeGuavaJDK5("com.google.apis"                % "google-api-services-sheets"  % "v4-rev494-1.23.0"),
+    excludeGuavaJDK5("com.google.apis"                % "google-api-services-cloudbilling" % "v1-rev14-1.23.0"),
```
Comment (davidangb) [inline project/Dependencies.scala:47]: I've updated all the google dependencies so they are all on the 1.23 version; previously we had mixed versions

**Comment by breilly2** [comment]
Nice! I may borrow this as I ran into something similar last week when I needed to update the version of Selenium we use. It would be great if we had a way to share this technique of importing Google libs sans guava-jdk5 in all of our projects.

**Review by rushtong — APPROVED** [review]
👍

**Pull request merged**